### DISCUSSION TITLE
Feature: 这是一个关于图片发太多QQ被腾讯风控的故事

### DISCUSSION THREAD

**Discussion by by-yitong** [koishijs/novelai-bot#139] [General]
### Scope

NovelAI

### Describe the problem related to the feature request

因为图片发太多QQ被腾讯风控

### Describe the solution you'd like

null

**Comment by MaikoTan** [comment]
This would not gonna happen until you write down a valid solution.

直到你写下一个合法的解决方案之前这不会发生。

**Comment by by-yitong** [comment]
> This would not gonna happen until you write down a valid solution.
> 
> 直到你写下一个合法的解决方案之前这不会发生。

6

**Comment by MaikoTan** [comment]
By the way, "Wind-control" is a complex problem on QQ platform, we could not just change some code and then call it a day. 'Cause the Tencent refused to provide a valid approach to bots on QQ at first.

顺便一提，“风控”在QQ平台上是一个复数的问题，我们不能改一些代码然后称它为一天。因为腾讯公司从一开始拒绝提供一个合法的途径到机器人在QQ上。

**Comment by shigma** [comment]
这不是一个可以解决的问题。因此将它挪到 discussion 以供讨论。
